src/js/view/controls/controls.js

```diff
@@ -368,8 +400,9 @@ export default class Controls {
     }
 
     userActive(timeout = ACTIVE_TIMEOUT, isKeyDown) {
```

Comment:
Please remove the default `timeout = ACTIVE_TIMEOUT`. We are already using `timeout || ACTIVE_TIMEOUT` which is less strict and doesn't get transpiled by babel into more code.